lib/ts/types.ts

```diff
@@ -65,6 +67,202 @@ export type TypeInput = {
     telemetry?: boolean;
     isInServerlessEnv?: boolean;
     debug?: boolean;
+    security?: {
+        googleRecaptcha?: {
+            // if the user provides both, we will use v2
+            v2SecretKey?: string;
+            v1SecretKey?: string;
+        };
+        override?: (
+            originalImplementation: SecurityFunctions,
+            builder?: OverrideableBuilder<SecurityFunctions>
+        ) => SecurityFunctions;
+    };
```

Comment:
we can ask the users to pass the api keys for the services here, and then tell the core that these are being used in there. This way, the core would know to send license key ping info.